models/DataObject/ClassDefinition/Helper/DefaultValueGeneratorResolver.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Pimcore\Model\DataObject\ClassDefinition\Helper;
+
+use Pimcore\Logger;
+use Pimcore\Model\DataObject\ClassDefinition\DefaultValueGeneratorInterface;
+
+class DefaultValueGeneratorResolver
```

Comment:
why not use `ClassResolver`?

Reply:
This is part of #5027 which hat not been merged yet.

Reply:
true, so we should wait for #5027 to be merged before this